cpp/src/aztec/transcript/transcript.cpp

```diff
@@ -15,6 +15,16 @@
 
 namespace transcript {
 
+// Set to 1 to enable debug logging.
```

Comment:
If you want to keep this around, would you put it in common/log.hpp (I think that's the right place). Also debug is a very generic name, would be better called `pretty_info` or something more specific.

Reply:
Yeah the problem is there's something similar in polynomial_cache (originally done by Charlie) so would want to be consistent with that - if I move it to a central location I'm not sure how you could independently toggle logging in one or the other. I'll change the name in both places to `info_togglable` or something and we can decide on a more general solution later?

Reply:
OK, you say it'll be useful in the short term, so I can see the value of keeping this around for now.